.brigade/brigade.ts

```diff
@@ -0,0 +1,224 @@
+import { events, Event, Job, ConcurrentGroup, SerialGroup, Container } from "@brigadecore/brigadier"
+
+const releaseStoragePath = "/release-assets"
+
+const goImg = "brigadecore/go-tools:v0.1.0"
```

Comment:
Should we bump to the latest release tag (v0.5.0)?  Or perhaps this is the best fit with the go version for v1?

Reply:
I'm unopposed to doing that in a separate PR. For now I was doing my best to replicate the the old process using Brigade 2 and without changing anything I didn't have to.